app/views/restaurants/_header.html.erb

```diff
@@ -0,0 +1,25 @@
+<!-- Shop basic info. name | distict | stars | street | price range -->
+<div class="row">
+  <div class="col-lg-12">
+      <h1 class="page-header">
+        <%= @restaurant.name %>
+<!--      Insert Ratings here
```

Comment:
remove commented code pls, you may add it when you implement the ratings feature later